### ISSUE TITLE
INFO: fetch-response is unable to open the file

### ISSUE THREAD

**Issue by cristobalfuenzalida** [mas-cli/mas#337]
Some mas-cli commands like `mas account` and `mas list`are working fine. But when I try to run `mas outdated` or `mas upgrade` I get the following error:
```
(base) ➜  ~ mas outdated
2021-03-25 04:15:53.204 mas[26314:2574999] INFO: fetch-response is unable to open the file /Users/<username>/Library/Caches/com.mphys.mas-cli/fsCachedData/0BC3AC3D-...-...-...-FB47D8BEEF7C. Errno: 2
(base) ➜  ~ mas upgrade
2021-03-25 04:15:53.204 mas[26314:2574999] INFO: fetch-response is unable to open the file /Users/<username>/Library/Caches/com.mphys.mas-cli/fsCachedData/0BC3AC3D-...-...-...-FB47D8BEEF7C. Errno: 2
Warning: Nothing found to upgrade
(base) ➜  ~
```
I would have to wait until one of my apps gets an update to check whether `mas upgrade` is actually working.
As recommended, I ran `mas reset` several times after restarting my machine and logging out and back into the App Store. Nothing worked.

I'm running macOS Big Sur 11.2.3 on a MacBook Pro (13-inch, 2017). I installed mas-cli via Homebrew, and the last time I used mas-cli was last week. Likewise, I run `brew update && brew upgrade`, and then `brew cleanup && brew doctor` every day, so maybe it's due to a recent update. But I always get `Your system is ready to brew.` when running the `doctor` command.

**Comment by phatblat** [comment]
Thanks for the bug report! I haven't seen this before but am able to reproduce it intermittently.

```
↪ mas upgrade
2021-03-28 19:20:28.230 mas[37135:9460737] INFO: fetch-response is unable to open the file /Users/phatblat/Library/Caches/com.mphys.mas-cli/fsCachedData/D26A3289-5825-46EB-8EB5-2972AAD48AF1. Errno: 2
```

I'm on macOS 11.2.3 (20D91) as well.

**Comment by EmilVarona** [comment]
I got this same error today for the first time. FWIW, I reinstalled mas using brew and still got the error. And I am also running Mac OS 11.2.3.

`coltrane:~$ mas outdated
2021-04-06 12:13:52.872 mas[58928:3102904] INFO: fetch-response is unable to open the file /Users/$USER/Library/Caches/com.mphys.mas-cli/fsCachedData/161D2E83-...E8C0E2E45236. Errno: 2
coltrane:~$ mas upgrade
2021-04-06 12:16:08.517 mas[59141:3106078] INFO: fetch-response is unable to open the file /Users/$USERLibrary/Caches/com.mphys.mas-cli/fsCachedData/161D2E83-...E8C0E2E45236. Errno: 2
Warning: Nothing found to upgrade`

**Comment by bumpux** [comment]
Same error here. No sign so far that it is interfering with mas-cli.

**Comment by toonetown** [comment]
I think it actually _is_ interfering with the mas-cli...particularly the `outdated` command.  There are many reports of the `outdated` command not showing correct information (or listing old versions as the "update" to new versions, etc).  These problems started occurring at the same time for me (and might be related to upgrading to Big Sur 11.2).

For example, if I open the failed which it stats is unreadable (it _is_ actually readable), it *ALWAYS* references the incorrectly-displayed `outdated` package.

Also - over the past month or so that I've been noticing this issue, the files that it fails to read has changed - as have the incorrectly-displayed `outdated` packages...but the file has always matched the outdated error.

**Comment by dynamicworkplace** [comment]
Im occasionally getting the "INFO: fetch-response is unable to open the file" error, and upgrade-apps is repeatedly downloading and installing the same apps (although it reports they are installed successfully each time). I've tried a `mas reset` and have uninstalled and reinstall mas using brew.

**Comment by chris-araman** [comment]
This should be fixed in #345 and is expected to be included in the next release.
